src/use/use.py

```diff
@@ -957,62 +920,56 @@ def _load_venv_entry(name, module_path) -> ModuleType:
 
 @cache(maxsize=512, typed=True)
```

Comment:
why maxsize? i'm betting a large system can easily hit that number..

Reply:
default wasn't big enough